src/contents/en-gb/home.yaml

```diff
@@ -46,6 +46,18 @@ contribute:
     link:
       name: Core Technology
       href: /core_technology
+sharing:
+  title:
+    shared: Stand alone
+    tech: technology
+    safe: SAFE Network
+    desc: Use standalone components from the SAFE Network in your own projects.
+  parsec:
+    title: PARSEC
+    desc: PARSEC (Protocol for Asynchronous, Reliable, Secure and Efficient Consensus) is an algorithm which allows a section of network nodes to reach consensus on the validity and order of network events by voting on events and using a gossip protocol to disseminate these votes amongst themselves.
```

Comment:
Sorry to be picky .. but this sentence needs some commas and/or full stops to help the reader digest the info. Or provide just a summary sentence here (e.g. PARSEC (Protocol for Asynchronous, Reliable, Secure and Efficient Consensus) is an algorithm which allows nodes to reach consensus in a decentalised network.), and provide the fuller explanation on the page.

Reply:
Na, it's fair comment.

Reply:
done